app/i18n/de/install.php

```diff
@@ -4,7 +4,9 @@
 	'action' => array(
 		'finish' => 'Installation fertigstellen',
 		'fix_errors_before' => 'Bitte Fehler korrigieren, bevor zum nächsten Schritt gesprungen wird.',
+		'keep_install' => 'Keep previous installation',  // TODO: translate
```

Comment:
You will lose your previous configuration by reinstalling FreshRSS. Are you sure you want to continue? --> Du wirst deine vorherige Konfiguration (Daten) verlieren FreshRSS. Bist du sicher, dass du fortfahren willst?